index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Project1</title>
+		<script src="https://kit.fontawesome.com/62fc5eef3c.js" crossorigin="anonymous"></script>
+	</head>
+	<body>
+		<!--Navbar -->
+		<header>
+			<nav>
+				<div class="nav-btns-left">
+					<p><a class="link" href="#about">About</a></p>
+					<p><a class="link" href="#portfolio">Portfolio</a></p>
+				</div>
+				<div class="nav-btn-right">
+						<p><a class="link" href="#contact">Contact</a></p>
+				</div>
+			</nav>
+		</header>
+	<!-- Main -->
+	 <main>
+		<div class="imgs-container">
+			<img src="kid.jpg" alt="kid statue">
+			<img src="london.jpg" alt="london eye image">
+			<img src="girl.jpg" alt="girl relaxing">
+			<img src="statue.jpg" alt="image of a statue">
+		</div>
+		<section class="hello-section">
+			<h1>Hello there!</h1>
+			<h2>Welcome to Cosmin's portfolio</h2>
+		</section>
+	</main>
+	<!--About section -->
+	<section class="about-section">
+		<h2 class="h3-about" id="about">About</h3>
+		<h3>Hi, I’m Cosmin , originally from Romania..I chose to come to the UK roughly 4 years ago to start a new chapter. Having just finished my master's degree in Geography in Romania, the oportunities to start something there after I finished my college, were quite limited. My life in the UK was like a rollercoaster in all this years, with a pandemy just gone..and kept switching between jobs ..I found myself now to be a student at iO academy, and I must admit I absolutely love it! I just hope that I will be able soon to fullfill my dream and become a programmer.
```

Comment:
i think this is more of a p than h3 in terms of its purpose, so you should change it to a p